resources/services/elasticbeanstalk/application_versions.go

```diff
@@ -0,0 +1,149 @@
+package elasticbeanstalk
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/elasticbeanstalk"
+	"github.com/aws/aws-sdk-go-v2/service/elasticbeanstalk/types"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/diag"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+//go:generate cq-gen --resource application_versions --config gen.hcl --output .
+func ApplicationVersions() *schema.Table {
+	return &schema.Table{
+		Name:         "aws_elasticbeanstalk_application_versions",
+		Description:  "Describes the properties of an application version.",
+		Resolver:     fetchElasticbeanstalkApplicationVersions,
+		Multiplex:    client.ServiceAccountRegionMultiplexer("elasticbeanstalk"),
+		IgnoreError:  client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter: client.DeleteAccountRegionFilter,
+		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"application_version_arn"}},
+		Columns: []schema.Column{
+			{
+				Name:        "account_id",
+				Description: "The AWS Account ID of the resource.",
+				Type:        schema.TypeString,
+				Resolver:    client.ResolveAWSAccount,
+			},
+			{
+				Name:        "region",
+				Description: "The AWS Region of the resource.",
+				Type:        schema.TypeString,
+				Resolver:    client.ResolveAWSRegion,
+			},
+			{
+				Name:        "application_name",
+				Description: "The name of the application to which the application version belongs.",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "application_version_arn",
+				Description: "The Amazon Resource Name (ARN) of the application version.",
+				Type:        schema.TypeString,
+			},
```

Comment:
would change to "arn"

Reply:
Makes sense, renamed the field and fixed the migrations